echo-webhooks/src/main/groovy/com/netflix/spinnaker/echo/scm/github/GithubPullRequestEvent.java

```diff
@@ -17,23 +17,21 @@
 package com.netflix.spinnaker.echo.scm.github;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.netflix.spinnaker.echo.api.events.Event;
-import java.util.Map;
 import java.util.Optional;
 import lombok.Data;
 
 @Data
 public class GithubPullRequestEvent implements GithubWebhookEvent {
-  Repository repository;
+  private Repository repository;
 
   @JsonProperty("pull_request")
-  PullRequest pullRequest;
+  private PullRequest pullRequest;
 
-  String action;
+  private String action;
 
   // `.repository.full_name`
   @Override
-  public String getFullRepoName(Event event, Map postedEvent) {
```

Comment:
Are these args not actually used in any of these methods?

Reply:
Nope, nowhere.